go/kbfs/libkbfs/folder_branch_ops.go

```diff
@@ -7947,6 +7950,7 @@ func (fbo *folderBranchOps) ClearPrivateFolderMD(ctx context.Context) {
 			fbo.cancelEdits = nil
 		}
 		fbo.editHistory = kbfsedits.NewTlfHistory()
+		fbo.launchEditMonitor = sync.Once{}
```

Comment:
If I were looking at this line outside of the context of this PR, I would be confused as to what it did / what the point was. Can you add a comment?